Dockerfile

```diff
@@ -15,6 +15,7 @@ COPY api ./api
 COPY cmd ./cmd
 COPY internal ./internal
 COPY main.go Makefile ./
+COPY data/resources ./data/resources
```

Comment:
We discussed extracting the resources to a separate image. Let's keep it here for now, until we have a use case to separate it! +1 to this